### PR TITLE
Add WCA Forum to the Source

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Original source of the WCA regulations: [www.worldcubeassociation.org/regulation
 WCA Regulations in [PDF format](link:pdf)
 
 ### Source
-Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/thewca/wca-regulations).
+Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/thewca/wca-regulations) and the discussion is public [on WCA Forum](https://forum.worldcubeassociation.org/c/regulations).
```
Comment (lgarron) [inline wca-regulations.md]:
```suggestion
Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/thewca/wca-regulations) and the discussion is public [on the WCA Forum](https://forum.worldcubeassociation.org/c/regulations).
```

**Comment by lgarron** [comment]
Thanks for drafting this!

If we want to update this between major updates, I think we should:

- Ask the Board for approval for a textual revision.
- Merge this into `official`.
- Merge `official` into `draft`.

**Comment by Ivan-Brigidano** [comment]
> Thanks for drafting this!
> 
> If we want to update this between major updates, I think we should:
> 
> * Ask the Board for approval for a textual revision.
> * Merge this into `official`.
> * Merge `official` into `draft`.

I don't think is strictly necessary to ask the Board because this is not actually a Regulations change. I will ask the Board just in case though.

What do you mean by "Merge `official` into `draft`"? If you make a Pull Request of changes from `official` into `draft` you will have some unwanted changes.

**Comment by lgarron** [comment]
> I don't think is strictly necessary to ask the Board because this is not actually a Regulations change. I will ask the Board just in case though.

Thanks!

> What do you mean by "Merge `official` into `draft`"? If you make a Pull Request of changes from `official` into `draft` you will have some unwanted changes.

Hmm, it looks like there was such a merge recently, so I think this shouldn't be much of an issue?

**Comment by Ivan-Brigidano** [comment]
> > What do you mean by "Merge `official` into `draft`"? If you make a Pull Request of changes from `official` into `draft` you will have some unwanted changes.
> 
> Hmm, it looks like there was such a merge recently, so I think this shouldn't be much of an issue?

Well there are two small commits. So we can do it and then creating a new PR with these small typos fixes.

**Pull request merged**